lib/chai/core/assertions.js

```diff
@@ -215,6 +215,16 @@ module.exports = function (chai, _) {
   Assertion.addChainableMethod('contains', include, includeChainingBehavior);
   Assertion.addChainableMethod('includes', include, includeChainingBehavior);
 
+  // TODO: Add jsdoc
+
+  function assertFail (msg) {
+    if (msg) flag(this, 'message', msg);
+    var obj = flag(this, 'object');
+    throw new chai.AssertionError(msg, {}, obj);
+  }
+
+  Assertion.addMethod('fail', assertFail);
+
```

Comment:
I was having issues adding `fail` directly to `expect` because it's a function. Are there any examples of how functions are directly added to expect anywhere else in the code base?